content/resources/github-token-authentication/index.md

```diff
@@ -1,5 +1,5 @@
 +++
-title = "Remoting into DCS"
+title = "GitHub token authentication"
```

Comment:
already changed this soI think causing merge conflict

Reply:
Resolved merge conflict by pulling changes into fork, then merging master back into the development branch.